mergify_engine/context.py

```diff
@@ -778,6 +778,13 @@ async def _get_consolidated_data(self, name: str) -> ContextAttributeType:
             comments, _ = await self.consolidated_reviews()
             return [r["user"]["login"] for r in comments if r["state"] == "COMMENTED"]
 
+        elif name == "check-success-or-neutral-or-pending":
```

Comment:
Do we want to add this in the documentation?

Reply:
No it's internal only.